src/main/java/Dog.java

```diff
@@ -19,6 +19,7 @@ public Dog(String name, String owner){
     public void dorun(){
         System.out.println(name+" is runing");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)